includes/class-wp-job-manager-shortcodes.php

```diff
@@ -272,6 +274,16 @@ public function job_dashboard( $atts ) {
 		return ob_get_clean();
 	}
 
+	/**
+	 * Filters the url from paginate_links to avoid multiple calls for same action in job dashboard
+	 *
+	 * @param string $link
+	 * @return string
+	 */
+	public function filter_paginate_links( $link ) {
+		return esc_url( remove_query_arg( [ 'action', 'job_id', '_wpnonce' ], $link ) );
```

Comment:
I don't think we need to `esc_url` for this, as it will be done after the filter completes.

Reply:
Done!